app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -479,6 +571,26 @@ class HomeActivityTest {
     return HomeActivity.createHomeActivity(context, profileId)
   }
 
+  private fun isEllipsized() = object : TypeSafeMatcher<View>() {
+    override fun describeTo(description: Description) {
+      description.appendText("with ellipsized text")
+    }
+    override fun matchesSafely(view: View): Boolean {
+      if (view !is TextView) {
+        return false
+      }
+      val textView: TextView = view
+      val layout: Layout = textView.layout
+      val lineCount = layout.lineCount
+      if (lineCount > 0) {
+        if (layout.getEllipsisCount(lineCount - 1) > 0) {
+          return true
+        }
+      }
+      return false
+    }
```

Comment:
```suggestion
      val layout: Layout = textView.layout
      val lineCount = layout.lineCount
      return lineCount > 0 && layout.getEllipsisCount(lineCount - 1) > 0
    }
```

or

```suggestion
      return with (textView.layout) { lineCount > 0 && getEllipsisCount(lineCount - 1) > 0 }
    }
```

Potential simplifications.

Reply:
Done